.trivyignore

```diff
@@ -1,5 +1,12 @@
-#telegraf MEDIUM
-GHSA-2w8w-qhg4-f78j
+#telegraf
+GHSA-2w8w-qhg4-f78j # MEDIUM
+CVE-2023-2253 # HIGH
+CVE-2023-28840 # MEDIUM
+CVE-2023-28841 # MEDIUM
```

Comment:
Did you get a chance to check if these vulns are fixed in latest version of telegraf and mariner has the version of this telegraf?